#! /bin/sh /usr/share/dpatch/dpatch-run
## no_dup_apps_icons.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not install application icons under /usr/share/f-spot.

@DPATCH@
diff -urNad trunk~/icons/Makefile.am trunk/icons/Makefile.am
--- trunk~/icons/Makefile.am	2007-12-04 12:19:26.000000000 +0000
+++ trunk/icons/Makefile.am	2007-12-25 14:45:47.000000000 +0000
@@ -54,11 +54,6 @@
 	actions,view-restore-22.png		\
 	actions,view-restore-24.png		\
 	actions,view-restore-32.png		\
-	apps,f-spot.svg				\
-	apps,f-spot-16.png			\
-	apps,f-spot-22.png			\
-	apps,f-spot-24.png			\
-	apps,f-spot-32.png			\
 	devices,camera-photo-16.png		\
 	devices,camera-photo-22.png		\
 	devices,camera-photo-24.png		\
